tests/generators/ssz_generic/ssz_container.py

```diff
@@ -105,7 +105,7 @@ def invalid_cases():
                          RandomizationMode.mode_one_count,
                          RandomizationMode.mode_max_count]:
                 if len(offsets) != 0:
```

Comment:
unrelated but we should be able to remove this line and outdent the body
```suggestion
            if len(offsets) != 0:
```

Reply:
fixed in #2791.